test/util/nightwatch-commands/selectRadio.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Selects the appropriate option for yesNo widgets.
+ *
+ * @param {String} fieldName The name of the field without Yes or No
```

Comment:
Love this JSDoc style comment, should we standardize this to automate api documentation?